### DISCUSSION TITLE
Can OpenPDF display Chinese characters?

### DISCUSSION THREAD

**Discussion by justinmulligan** [LibrePDF/OpenPDF#805] [Q&A] [answered]
Hi

I have an app that can be used in Chinese and i want the data from the Room DB to be exported to PDF using OpenPDF. Currently when OpenPDF gets to a field that has Chinese characters in it, the field on the actual exported PDF is blank.

So can OpenPDF output Chinese characters? Am i missing something?

Kind regards

**Comment by asturio** [comment, marked as answer]
It's important do use a font supporting Chinese. Embbeding the font should help. Take a look at some examples using non-latin.
Take a look at:

pdf-toolbox/src/main/java/org/librepdf/openpdf/examples/fonts/languages/Chinese.java
or
pdf-toolbox/src/test/java/com/lowagie/examples/fonts/getting/ChineseJapaneseKorean.java

**Reply by justinmulligan** [reply]
Thanks for that. 

I had trouble using the first example here - pdf-toolbox/src/main/java/org/librepdf/openpdf/examples/fonts/languages/Chinese.java

but got it working using this example - pdf-toolbox/src/test/java/com/lowagie/examples/fonts/getting/ChineseJapaneseKorean.java
